code/datums/dna.dm

```diff
@@ -18,21 +18,23 @@
 	var/scrambled = FALSE //Did we take something like mutagen? In that case we cant get our genes scanned to instantly cheese all the powers.
 	var/current_body_size = BODY_SIZE_NORMAL
 	//Holder for the displacement appearance, related to species height
-	var/icon/height_displacement
+	var/atom/movable/height_displacement_holder
 
 /datum/dna/New(mob/living/new_holder)
 	if(istype(new_holder))
 		holder = new_holder
-	height_displacement = icon('icons/effects/64x64.dmi', "height_displacement")
+	update_height_holder()
+	new_holder?.vis_contents += height_displacement_holder
```

Comment:
To be honest, I'm not really a fan of this change since its a little bit janky and breaks the right click menu. Does the monkey height displacement really have to be directional?

I would like to note that displacement maps do not update the hitbox for a sprite.

The level of jankiness here kind of feels like it offsets the advantages of converting monkeys into a species in the first-place and is something I feel like definitely needs some action to be taken. The most optimal solution would be finding some kind of displacement map that can just work for all directions in a way thats good enough. Shrinking the sprite vertically is always going to make it look a bit ugly just because its quite a big restriction on how monkeys can look. If there is absolutely nothing that can be done then... I guess we kind of have to live with it a bit.

If absolutely nothing can be done, and we have to have a vis_contents displacement map that has to respond to directions then does the map have to be 64x64 instead of 32x32? If it could be 32x32 then the alpha could be set to 0 which would limit its effect on the right click menu.

Reply:
>  Does the monkey height displacement really have to be directional?
Yes, it also opens up future stuff for other species

Reply:
> Yes, it also opens up future stuff for other species.

![image](https://github.com/user-attachments/assets/4459d8c6-a2bc-4bef-b47b-60b86e374b4b)

If we want to do unique stuff with height displacement, I think this is a good idea.

Having an excessively short race other than monkeys would make sense, given... basically every race is within the same margin as regular humans, which is rather anthropocentric... 

See Shiptest's short dinosaurian-bird race for this done fairly well.

Reply:
You might even be able to revive Gr*d with this new approach